src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -11,8 +11,10 @@ MPL_SUPPRESS_OSX_HAS_NO_SYMBOLS_WARNING;
 #ifdef MPL_HAVE_ZE
 
 ze_driver_handle_t global_ze_driver_handle;
-ze_device_handle_t *global_ze_devices_handle;
-int gpu_ze_init_driver();
+ze_device_handle_t *global_ze_devices_handle = NULL;
```

Comment:
Can we add a comment at the beginning of this file, clarifying that we are supporting level zero API v1.0? Is there a version check API to confirm that we are using the correct ze library?

Reply:
Level zero API is still evolving. At this time, we will always support the latest version and not maintaining the backward compatibility.  In v1.0, even the VERSION string macro changed into a runtime checking (via a function new function API), so there is no reliable/stable version check API at this time.

Reply:
I see. That's fine. Can we at least add a comment (at the beginning of this file) on the API version and a known ze library build number/so version (there must be something that we can use to identify the library, right?)?

Reply:
done